### PR TITLE
add missing provision.sh

### PR DESCRIPTION
**Pull request by AlCutter**
This should probably be updated to use the `createTree` command at some point.

### CODE REVIEW

File: examples/deployment/kubernetes/provision_tree.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+echo "Created tree ${TREEID}:"
+echo ${TREE} | jq
+
+echo "Initialising tree ${TREEID}:"
+STH=$(curl -s -X POST ${LOG_URL}/v1beta1/logs/${TREEID}:init)
+echo "Created STH:"
+echo ${STH} | jq
+
+kill %1
+
```
Comment (daviddrysdale) [inline examples/deployment/kubernetes/provision_tree.sh]: lots of blank lines

File: examples/deployment/kubernetes/provision_tree.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+echo "Created tree ${TREEID}:"
+echo ${TREE} | jq
+
+echo "Initialising tree ${TREEID}:"
+STH=$(curl -s -X POST ${LOG_URL}/v1beta1/logs/${TREEID}:init)
+echo "Created STH:"
+echo ${STH} | jq
+
+kill %1
```
Comment (daviddrysdale) [inline examples/deployment/kubernetes/provision_tree.sh]: %1 is some time ago -- a comment (`# stop port forwarding`) would be nice

File: examples/deployment/kubernetes/provision_tree.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+PORT=35791
+export LOG_URL="http://127.0.0.1:${PORT}"
+
+echo kubectl port-forward service/trillian-log-service ${PORT}:8091
+kubectl port-forward service/trillian-log-service ${PORT}:8091 &
+trap "kill %1" 0 1 2 3 4 6 9 15
+
+sleep 3
+
+echo "Creating tree..."
```
Comment (daviddrysdale) [inline examples/deployment/kubernetes/provision_tree.sh]: Add `TODO(alcutter): use createtree instead` ?

File: examples/deployment/kubernetes/provision_tree.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+TREE=$(curl -sb -X POST ${LOG_URL}/v1beta1/trees -d '{ "tree":{ "tree_state":"ACTIVE", "tree_type":"LOG", "hash_strategy":"RFC6962_SHA256", "signature_algorithm":"ECDSA", "max_root_duration":"0", "hash_algorithm":"SHA256" }, "key_spec":{ "ecdsa_params":{ "curve":"P256" } } }')
+if [ $? -ne 0 ]; then
+  echo "Failed to create tree:"
```
Comment (daviddrysdale) [inline examples/deployment/kubernetes/provision_tree.sh]: Can you hit this arm, given the `set -e` ?

File: examples/deployment/kubernetes/provision_tree.sh (hunk 3 above)
Comment (AlCutter) [reply examples/deployment/kubernetes/provision_tree.sh]: added

File: examples/deployment/kubernetes/provision_tree.sh (hunk 2 above)
Comment (AlCutter) [reply examples/deployment/kubernetes/provision_tree.sh]: done

File: examples/deployment/kubernetes/provision_tree.sh (hunk 1 above)
Comment (AlCutter) [reply examples/deployment/kubernetes/provision_tree.sh]: dinged

File: examples/deployment/kubernetes/provision_tree.sh (hunk 4 above)
Comment (AlCutter) [reply examples/deployment/kubernetes/provision_tree.sh]: Good spot, removed.

**Pull request merged**